script.js

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable no-plusplus */
+const $books = document.querySelector('.books');
+const $title = document.querySelector('.title');
+const $author = document.querySelector('.author');
+const $addButton = document.querySelector('.addButton');
+
+document.addEventListener('DOMContentLoaded', () => {
+  class Books {
+    constructor() {
+      this.bookList = JSON.parse(localStorage.getItem('bookList')) || [];
+      this.arr = [];
```

Comment:
- [ ] It would be nice if you have an array of books outside of the class to follow the project requirements :+1: In this project, you are not supposed to implement classes because it will be in your next project :+1:

Reply:
@HAMMAS-SALEEM Done it, the array declaration is now according to the requirements.